roles/ceph-docker-common/tasks/fetch_image.yml

```diff
@@ -274,27 +274,56 @@
     - ceph_nfs_container_inspect_before_pull.get('rc') == 0
     - ceph_nfs_image_repodigest_before_pulling != image_repodigest_after_pulling
 
+- name: make local tempdir for ceph dev images
+  local_action:
+    module: tempdir
+    state: directory
+    prefix: ceph_ansible
+  register: localtmpdir
+  when:
+    - (ceph_docker_dev_image is defined and ceph_docker_dev_image)
+  run_once: true
+
+- name: make target tempdir for ceph dev images
+  tempdir: 
```

Comment:
typo?

Reply:
Yes, sorry; pushed a fix (squashed into the previous commit)